cmd/xl-v1-utils.go

```diff
@@ -99,19 +101,160 @@ func hashOrder(key string, cardinality int) []int {
 	return nums
 }
 
+func parseXLStat(xlMetaBuf []byte) (statInfo, error) {
+	// obtain stat info.
+	stat := statInfo{}
+	// fetching modTime.
+	modTime, err := time.Parse(time.RFC3339, gjson.Get(string(xlMetaBuf), "stat.modTime").String())
```

Comment:
It may be worth noting that gjson recently added a `GetBytes` function that allows for directly working a byte slices.  

```go
modTime, err := time.Parse(time.RFC3339, gjson.GetBytes(xlMetaBuf, "stat.modTime").String())
```
While it will only give a modest bump (5-10%) in performance, it does avoid the []byte->string reallocation, which can add up if you are using large json objects.